src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx

```diff
@@ -10,7 +10,7 @@ import { HowItWorksSteps } from "Apps/Consign/Routes/MarketingLanding/Components
 // import { MeetTheSpecialists } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists"
 import { FAQSWA } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/FAQSWA"
 import { Footer } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Footer"
-import { Reviews } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Reviews"
+// import { Reviews } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Reviews"
```

Comment:
**Suggestion:** We should remove them rather than comment them out. We can always bring them back when we look at the PR that removed them. Commented out code tends to stay for a very long time.